### PR TITLE
Esp32 support

### PR DESCRIPTION
**Pull request by ofekp**

### CODE REVIEW

**Comment by ofekp** [comment]
@GIPdA I will test this with my ESP32 soon and we'll get it merged if that's OK with you.
This is really amazing :) Thank you!

**Comment by GIPdA** [comment]
No problem, please merge if your tests succeed :)
I intended to make a pull request at one point but I forgot :P

File: src/TinyUPnP.cpp [hunk 1]
```diff
@@ -697,25 +721,30 @@ boolean TinyUPnP::getIGDEventURLs(gatewayInfo *deviceInfo) {
 				urlBaseFound = true;
 			}
 		}
+
+		// to support multiple <serviceType> tags
```
Comment (ofekp) [inline src/TinyUPnP.cpp]: @GIPdA can you please explain why you needed this?

Comment (ofekp) [reply src/TinyUPnP.cpp]: ➡️ Moving over to https://github.com/ofekp/TinyUPnP/pull/42 instead since it has a branch.

**Comment by ofekp** [comment]
➡️ Moving over to https://github.com/ofekp/TinyUPnP/pull/42 instead since it has a branch.

**Pull request closed without merging**